### PR TITLE
Fix loadViewsFrom compatibility (Lumen / PHP 7.4)

### PR DESCRIPTION
**Pull request by kylekatarnls**

### CODE REVIEW

File: src/Illuminate/Support/ServiceProvider.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ protected function loadRoutesFrom($path)
      */
     protected function loadViewsFrom($path, $namespace)
     {
-        if (is_array($this->app->config['view']['paths'])) {
+        if (is_array($this->app->config['view']) && is_array($this->app->config['view']['paths'])) {
             foreach ($this->app->config['view']['paths'] as $viewPath) {
```
Comment (driesvints) [inline src/Illuminate/Support/ServiceProvider.php]: Maybe typecast to an array?

Comment (kylekatarnls) [reply src/Illuminate/Support/ServiceProvider.php]: A test is more optimized than a typecast. Nope?

Comment (driesvints) [reply src/Illuminate/Support/ServiceProvider.php]: I meant:

```suggestion
            foreach ((array) $this->app->config['view']['paths'] as $viewPath) {
```

Comment (kylekatarnls) [reply src/Illuminate/Support/ServiceProvider.php]: The error is not about the type of `$x['paths']`, PHP 7.4 raise an error if `config['view']` is undefined (it happens when 'view' is not configured).

Comment (driesvints) [reply src/Illuminate/Support/ServiceProvider.php]: But that will throw an notice regardless? https://3v4l.org/cISbX

Comment (kylekatarnls) [reply src/Illuminate/Support/ServiceProvider.php]: Right, `is_array` is not good, what we need is `isset` and as it's always an array when it's set, the `isset` will be enough.

**Comment by driesvints** [comment]
Perfect. Thanks.

**Pull request merged**